agent/functional_tests/testdata/taskdefinitions/task-shared-vol-read/task-definition.json

```diff
@@ -0,0 +1,36 @@
+{
+    "family": "ecsftest-task-local-volume",
+    "containerDefinitions": [
+        {
+            "name": "task-shared-vol-read",
+            "image": "127.0.0.1:51670/busybox:latest",
+            "cpu": 10,
+            "command": ["sh", "-c", "while true; do sleep 1; [ -f /ecs/success ] && if grep -q 'can you read me' /ecs/success; then exit 42; fi done"],
+            "memory": 256,
+            "memoryReservation": 128,
+            "mountPoints": [
+                {
+                  "sourceVolume": "task-shared",
+                  "containerPath": "/ecs/"
```

Comment:
task.Cleanup should take care of it, double checking.

Reply:
task scoped volume should be cleaned up, and this is covered by integ test Peng wrote.

Reply:
what about the other scope? if not, it could lead to false positives if you run it repeatedly on the same host.

Reply:
shared scope volumes are never meant to be deleted by task spin down.

Reply:
should our test runner clean it up then?

How do we know the test succeeded and isn't relying on a previous artifact?

Reply:
it's not the testing logic that's doing that, it's what "shared" volume is supposed to be like. Because it's shared, agent doesn't take part in cleaning it up.